src/net/sourceforge/kolmafia/persistence/FaxBotDatabase.java

```diff
@@ -440,6 +449,20 @@ public void run() {
       FaxBotDatabase.faxbots.addAll(bots);
     }
 
+    private static final String NOHASH = "Problem with hash";
+
+    private static String computeHash(File file) {
+      String checksum;
+      try {
+        byte[] data = Files.readAllBytes(file.toPath());
+        byte[] hash = MessageDigest.getInstance("MD5").digest(data);
```

Comment:
Can we pick a hash algorithm that doesn't have known collision attacks? SHA-256 should be fine, or if you want to be really safe, SHA3-512 should also be broadly available.

Reply:
Done